libshaderc/src/shaderc.cc

```diff
@@ -116,16 +116,18 @@ class StageDeducer {
   // 'default' shader kind to the corresponding shader stage. As this is the
   // last trial to determine the shader stage, failing to find the corresponding
   // shader stage will cause an error.
-  EShLanguage operator()(std::ostream* error_stream,
-                         const shaderc_util::string_piece& error_tag) const {
+  EShLanguage operator()(std::ostream* /*error_stream*/,
+                         const shaderc_util::string_piece& /*error_tag*/) {
     EShLanguage stage = GetDefaultStage(kind_);
     if (stage == EShLangCount) {
-      *error_stream << "error: '" << error_tag
-                    << "' could not determine the shader stage" << std::endl;
+      error_ = true;
     }
     return stage;
```

Comment:
Feels like we should set error_ = false in the ok case.
What is supposed to happen if glslang calls it twice with two different tags?   Ok, might not happen now, but we should make it well defined.